README.md

```diff
@@ -23,17 +25,11 @@ go get github.com/libp2p/go-floodsub
 
 ## Usage
 
-```
-TODO
-```
+To be used for messaging in p2p instrastructure (as part of libp2p) such as IPFS, Ethereum, other blockchains, etc.
 
 ## Implementations
 
-See [this issue](https://github.com/libp2p/go-floodsub/issues/77) for an introduction to pubsub, gossipsub and floodsub, which is in the repo for go-floodsub. A PR for gossipsub with Go is [here](https://github.com/libp2p/go-floodsub/pull/67); see [gerbil-simsub](https://github.com/vyzo/gerbil-simsub) for a high-level literate specification, as well as [the libp2p spec for gossipsub](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/README.md).
-
-The rust-libp2p implementation of floodsub is [here](https://github.com/libp2p/rust-libp2p/search?utf8=%E2%9C%93&q=floodsub&type=). Work is in progress on a gossipsub implementation for Rust with @jamesray1 from Drops of Diamond (which is developing sharding for Ethereum).
-
-Here is a [Javascript](http://github.com/libp2p/js-libp2p-floodsub) implementation of floodsub.
+See [libp2p/specs/pubsub/gossipsub#implementation-status](https://github.com/libp2p/specs/tree/master/pubsub/gossipsub#implementation-status) and [libp2p/specs/pubsub#Implementations](https://github.com/libp2p/specs/tree/master/pubsub#Implementations).
```

Comment:
Isn't just the link to `pubsub#Implementation` sufficient?

Reply:
After https://github.com/libp2p/specs/pull/103, yes.